### PR TITLE
Con 3672 v1 bulk upload new format

### PR DESCRIPTION
**Pull request by VasanthaKasirajan3008**

### CODE REVIEW

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Models/BulkUpload/CsvRecordMap.cs [hunk 1]
```diff
@@ -2,11 +2,23 @@
...
+            Map(x => x.GivenNames);
+            Map(x => x.DateOfBirth);
+            Map(x => x.StdCode);          
+            Map(x => x.StartDate);
+            Map(x => x.EndDate);
+            Map(x => x.TotalPrice);
+            Map(x => x.EPAOrgID);
+            Map(x => x.ProviderRef);
+            Map(x => x.AgreementId);
+            Map(x => x.EmailAddress);
```
Comment (hkf-tech) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Models/BulkUpload/CsvRecordMap.cs:21]: Is this mandatory?

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/BulkUpload/BulkUploadValidator.cs [hunk 2]
```diff
@@ -59,10 +59,10 @@ public IEnumerable<UploadError> ValidateFileSize(HttpPostedFileBase attachment)
             var apprenticeshipUploadModels = records as ApprenticeshipUploadModel[] ?? records.ToArray();
             if (!apprenticeshipUploadModels.Any()) return new[] { new UploadError(ApprenticeshipFileValidationText.NoRecords) };
 
-            if (apprenticeshipUploadModels.Any(m => m.CsvRecord.CohortRef != apprenticeshipUploadModels.First().CsvRecord.CohortRef))
+            if (apprenticeshipUploadModels.Any(m => m.CsvRecord.CohortRef != apprenticeshipUploadModels.First().CsvRecord.CohortRef && apprenticeshipUploadModels.First().CsvRecord.CohortRef != string.Empty))
```
Comment (hkf-tech) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/BulkUpload/BulkUploadValidator.cs:62]: do we need this?

**Review by hkf-tech — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/BulkUpload/BulkUploadValidator.cs (hunk 2 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/BulkUpload/BulkUploadValidator.cs]: Good point , removed :-)

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Models/BulkUpload/CsvRecordMap.cs (hunk 1 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.ProviderApprenticeshipsService.Web/Models/BulkUpload/CsvRecordMap.cs:21]: Yes, it is

**Review by hkf-tech — APPROVED** [review]

**Pull request closed without merging**